notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2988 @@
+{
```

Comment:
<p>I would genuinely be interested to hear an explanation of this (and we should make a comment on this, if only to address our "what do we see?"). </p><p> </p><p>The "spread" improves once we update the lower bound, but there's still a level of "scatter" on the low period side of the graph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>